server/server/error_handler.js

```diff
@@ -1,11 +1,34 @@
 // This is a general purpose error handler for the backend,
 // when the backend encounters errors this handler redirects to an error page where necessary next steps can be taken.
 
+const DBHandler = require("./database/db");
+const DBCFG = require("./database/db_config");
```

Comment:
Make this variable name consistent with other files
```suggestion
const DB_CONFIG = require("./database/db_config");
```